WalletWasabi.Fluent/ViewModels/NavBar/NavBarViewModel.cs

```diff
@@ -72,15 +73,11 @@ public void Activate()
 			})
 			.Subscribe();
 
-		this.WhenAnyValue(x => x.SelectedWallet)
-			.Select(x => x?.WalletViewModel?.WalletModel)
-			.BindTo(this, x => x.SelectedWalletModel);
-
 		this.WhenAnyValue(x => x.SelectedWalletModel)
 			.Select(model => Wallets.FirstOrDefault(w => w.WalletModel == model))
 			.BindTo(this, x => x.SelectedWallet);
 
-		SelectedWallet = Wallets.FirstOrDefault(x => x.WalletModel.Name == UiContext.WalletRepository.DefaultWalletName);
+		SelectedWallet = Wallets.FirstOrDefault(x => x.WalletModel.Name == UiContext.WalletRepository.DefaultWalletName) ?? Wallets.First();
```

Comment:
First() would crash if the list is empty (i.e.: Fresh install), probably FirstOrDefault() what we need.

Reply:
Done [3b55c08](https://github.com/zkSNACKs/WalletWasabi/pull/11937/commits/3b55c08d8c8d2912269d2fc2cd04f41c14be0eb5)